packages/typespec-ts/src/index.ts

```diff
@@ -459,3 +459,8 @@ export async function createContextWithDefaultOptions(
     context.program.emitters[0]?.metadata.name ?? "@azure-tools/typespec-ts"
   )) as SdkContext;
 }
+
+function isArm(context: EmitContext<Record<string, any>>) {
```

Comment:
Could be replaced with tcgc helper https://github.com/Azure/typespec-azure/issues/1794